tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.py

```diff
@@ -0,0 +1,139 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+import PointwiseFunctions.AnalyticSolutions.RelativisticEuler.FishboneMoncriefDisk as fm_disk
```

Comment:
You can always break the line with `\`s.

Reply:
To be concrete, from a brief test I did on my laptop you could do:
```python
import PointwiseFunctions.AnalyticSolutions.\
    RelativisticEuler.FishboneMoncrief as fm_disk
```